pages/newsletters/index.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div class="vx-row justify-center w-full" id="newsletter-container">
+    <VxCard class="mb-6 w-24 min-w-full">
+      <template slot="no-body">
+        <div
+          class="text-white flex flex-col justify-center items-center background-cover py-10"
+          :style="
+            backgroundGradient(
+              'https://cdn.discordapp.com/attachments/738198413987938464/744678215406321674/mountains-moon-trees-minimalism-hd.jpg'
+            )
+          "
+        >
+          <div class="w-full vx-row justify-center">
+            <img
+              :src="require('~/assets/images/quentotypeface.png')"
+              class="lg:w-1/3 w-5/6 md:w-1/2 xl:w-1/4 center"
+            />
+          </div>
+          <div class="text-white text-title text-6xl">Quento Blog</div>
+        </div>
+      </template>
+    </VxCard>
+    <NewsletterModal></NewsletterModal>
+
+    <div v-if="loaded" class="w-full lg:w-1/2">
+      <VxCard
+        v-for="(item, index) in newsletters"
+        :key="index"
+        class="mb-6 w-24 min-w-full"
+      >
+        <template slot="no-body">
+          <nuxt-link :to="`/newsletters/${item.id}`">
+            <div
+              class="text-white flex flex-col justify-between items-center background-cover p-10 h-full"
+              :style="backgroundGradient(item.backgroundImageUrl)"
+              style="min-height: 20vw;"
+            >
+              <div
+                class="text-white mb-4 text-ginger-b text-4xl w-full"
+                style="line-height: 1.125;"
+              >
+                {{ item.title }}
+              </div>
+              <div class="vx-row w-full" style="">
+                <vs-avatar class="icon">
+                  <img v-if="item.authorPhotoUrl" :src="item.authorPhotoUrl" />
+                  <template #text v-else>
+                    {{ item.authorDisplayName }}
+                  </template>
+                </vs-avatar>
+                <div class="vx-row w-3/4 flex-wrap pl-4">
+                  <div
+                    class="w-full text-ginger truncate font-semibold"
+                    style=""
+                  >
+                    {{ item.authorDisplayName }}
+                  </div>
+                  <div class="w-full text-ginger truncate font-light" style="">
+                    {{ item.createdAt.toDateString() }}
+                  </div>
+                </div>
+              </div>
+            </div>
+          </nuxt-link>
+        </template>
+      </VxCard>
+    </div>
+    <!-- <vs-button warn @click="toggleNotesModal(true)" class="w-full">
```

Comment:
commented code, get rid of it

Reply:
we need that for adding posts - not a better way to do it right now

Reply:
When I have time I'll write an admin only site as well but for nwo its fine